app/src/main/java/xyz/malkki/neostumbler/CoverageParams.kt

```diff
@@ -0,0 +1,6 @@
+package xyz.malkki.neostumbler
+
+data class CoverageParams(
+    val tileJson: String?,
+    val layerId: String?,
```

Comment:
Is this layer ID necessary? From UX perspective it might be kind of confusing, especially for non-technical users. I think we can just use all layers specified in the TileJSON?

Reply:
Layer ID should be used as source layer. I will change the code to parse it from the TileJSON.

Reply:
That would definitely make it more usable :+1: Maybe we can add all layers from the TileJSON? Then in the future if there are multiple layers (e.g. different ones for Wi-Fi coverage and cell tower coverage), they would work without changes

Reply:
Done via https://github.com/mjaakko/NeoStumbler/compare/e833de128188e3d29cbcf9d0b0a81384b0873c55..ba63cd755621c804af990f292df3d7f8e11598cb